.github/workflows/README.md

```diff
@@ -48,6 +48,7 @@ If packages are specified, only these packages will be built. If no packages are
 - Currently, this command only supports GitHub-hosted runners.
   - Microsoft Azure's Standard_DS2_v2 virtual machines only support x86-64 (`amd64`) architecture and may not have sufficient performance or storage to build some packages.
   - GitHub-hosted runners currently provide no TTY device[^1], any build that relies on this will fail.
+- At most 256 packages can be built at a time.

```

Comment:
Up to 256 packages could be built at a time.

Reply:
```suggestion
- Up to 256 packages could be built at a time.
```